File: C99/1000-1099/1070-1079/1074.c
#include <stdio.h>

int main()
{
    int n, num;
    scanf("%d", &n);
    if (n < 10000)
    {
        for (int i = 0; i < n; i++)
        {
            scanf("%d", &num);
            if (num == 0)
            {
                printf("NULL\n");
            }
            else if (num % 2 == 0)
            {
                if (num > 0)
                {
                    printf("EVEN POSITIVE\n");
                }
                else
                {
                    printf("EVEN NEGATIVE\n");
                }
            }
            else
            {
                if (num > 0)
                {
                    printf("ODD POSITIVE\n");
                }
                else
                {
                    printf("ODD NEGATIVE\n");
                }
            }
        }
    }
    return 0;
}

File: beecrowd/1000-1099/1070-1079/1074.py
num = int(input())
for n in range(num):
    entry = int(input())
    if entry == 0:
        print("NULL")
    elif entry > 0:
        if entry % 2 == 0:
            print("EVEN POSITIVE")
        else:
            print("ODD POSITIVE")
    else:
        if entry % 2 == 0:
            print("EVEN NEGATIVE")
        else:
            print("ODD NEGATIVE")